lib/travis/build/addons/deploy/script.rb

```diff
@@ -134,17 +134,13 @@ def run
               end
             end
 
-            def install(edge = config[:edge])
-              edge = config[:edge]
-              if edge.respond_to? :fetch
-                src = edge.fetch(:source, 'travis-ci/dpl')
-                branch = edge.fetch(:branch, 'master')
-                build_gem_locally_from(src, branch)
+            def install
+              sh.if "$(rvm use $(travis_internal_ruby) do ruby -e \"puts RUBY_VERSION\") = 1.9*" do
+                cmd(dpl_install_command(true), echo: false, assert: !allow_failure, timing: true)
```

Comment:
Nit: Could we consider not passing a bare `true` here? Either pass a variable with a more descriptive name e.g.

```
installDplPre1_9 = true
cmd(dpl_install_command(installDplPre1_9)...
```

Or pass the version to install instead e.g.

```
cmd(dpl_install_command('1.8')...
```

In this case the function could also be renamed to something like `dpl_install_version_command(version = 'latest')`

My 2¢